modules/flowable-engine/src/test/java/org/flowable/engine/test/jobexecutor/AsyncExecutorTest.java

```diff
@@ -54,28 +56,25 @@ public void testRegularAsyncExecution() {
 
             // Move clock 3 minutes. Nothing should happen
             addSecondsToCurrentTime(processEngine, 180L);
-            try {
-                waitForAllJobsBeingExecuted(processEngine, 500L);
-                Assert.fail();
-            } catch (FlowableException e) {
-                // Expected
-            }
-            Assert.assertEquals(1, processEngine.getTaskService().createTaskQuery().taskName("The Task").count());
-            Assert.assertEquals(0, processEngine.getTaskService().createTaskQuery().taskName("Task after timer").count());
-            Assert.assertEquals(1, processEngine.getManagementService().createTimerJobQuery().count());
-            Assert.assertEquals(0, getAsyncExecutorJobCount(processEngine));
+            ProcessEngine processEngine1 = processEngine;
```

Comment:
Hi @dbmalkovsky , can you elaborate a bit why the assignment to processEngine1 was added?

Reply:
@tijsrademakers the reason is that the `assertThatThrownBy` requires that the variables be effectively final in the clause and this was the easiest way to get that without changing a lot of code.